config/checkstyle/suppressions.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+        "-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN"
+        "https://checkstyle.org/dtds/suppressions_1_2.dtd">
+
+<suppressions>
+
+    <suppress checks="MagicNumberCheck" files=".*Test\.java"/>
```

Comment:
Can you specify the test directory here, instead of relying on regex? `src/test/.*`